lib/adapters/event-handling/testplane/snapshots.ts

```diff
@@ -92,7 +92,48 @@ export const finalizeSnapshotsForTest = async ({testResult, attempt, reportPath,
             return [];
         }
 
+        if (testResult.history && testResult.history.length > 0 && snapshots.length > 0) {
```

Comment:
Why is this needed? Consider the following case:
- History steps start from time 00:00:00 and end at 00:05:00
- Snapshots are available for range 00:01:00 – 00:04:30

This has two consequences: firstly, player timeline has shorter time than actual test steps, secondly, some steps are out of range completely.

Here, we add fake snapshots to the beginning and the end and fix this.